install.py

```diff
@@ -4,49 +4,54 @@
 
 
 def main():
-    package_command = 'sam package --template-file template.yaml --output-template-file deploy.yaml'
-    deploy_command = 'sam deploy --template-file deploy.yaml --capabilities CAPABILITY_NAMED_IAM '
+    package_command_list = ['sam', 'package', '--template-file', 'template.yaml', '--output-template-file', 'deploy.yaml']
+    deploy_command_list = ['sam', 'deploy', '--template-file', 'deploy.yaml', '--capabilities', 'CAPABILITY_NAMED_IAM']
     s3_bucket = input('Enter the s3 bucket name created as part of pre-requisite: ')
     if s3_bucket:
-        package_command = package_command + ' --s3-bucket ' + s3_bucket
+        package_command_list.append('--s3-bucket')
+        package_command_list.append(s3_bucket)
     region = input('Enter the region [for instance, us-west-2]: ')
     if region:
-        package_command = package_command + ' --region ' + region
-        deploy_command = deploy_command + ' --region ' + region
+        package_command_list.append('--region')
+        package_command_list.append(region)
+
+        deploy_command_list.append('--region')
+        deploy_command_list.append(region)
         print()
         print ('packaging trapheus for use ...')
         try:
-            subprocess.run(package_command, shell=True, check=True)
+            subprocess.run(package_command_list, shell=False, check=True)
```

Comment:
can we extract the try and except block into a separate method , something liek ```run_subprocess(command_list)```

Reply:
This extraction to further help commands to be executed in https://github.com/intuit/Trapheus/issues/169